apps/web/modules/ui/components/sidebar/index.test.tsx

```
@@ -0,0 +1,584 @@
+      render(
+        <SidebarProvider>
+          <SidebarTrigger onClick={customOnClick} />
+          <SidebarRail />
+          <TestComponent />
+        </SidebarProvider>
+      );
+
+      // Test SidebarTrigger
+      const trigger = screen.getByTestId("columns2-icon").closest("button");
+      expect(trigger).toBeInTheDocument();
+      await user.click(trigger!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)